packages/cursorless-engine/src/core/Cheatsheet.ts

```diff
@@ -4,6 +4,7 @@ import produce from "immer";
 import { sortBy } from "lodash";
 import { ide } from "../singletons/ide.singleton";
 import path from "path";
+import { CheatsheetInfo } from "@cursorless/cheatsheet";
```

Comment:
I'd add these to `packages/common` and import them from there. Having extension depend on a React component is probably ok, but we can avoid having that conversation by just moving the type to a shared dependency